### ISSUE TITLE
Fix kern classes

### ISSUE THREAD

**Issue by mjlagattuta** [VivaRado/Advent#8]
The current ufo for the Roman Regular at commit [`13c2320`](https://github.com/VivaRado/Advent/tree/13c2320c238ec99199758cd8a8c6c030ed148e4f) has 16,383 kerning pairs. I would think this number should be much closer to 2000 or 3000 max given the character set.

If the kerning classes were to be optimized or potentially manually set, I think this number could be improved. A lot of glyphs take on their own class though they should be in a more general class like "H" or "O" (i.e. D/D should instead be H/O)

**Comment by VivaRado** [comment]
Currently I'm trying to reduce by also rounding the values. Will send updates on kerning interpolation too.
I don't know how and if it is ever going to hit 3000 but that would be splendid!

At the moment: 
 - 'Group to Group': 3026, 
 - 'Group to Letter': 4141
 - 'Letter to Group': 4032
 - 'Letter to Letter': 5184

Total Pairs: 16383

Some ideas is partial with left and right combinations - I dont know how much will that save but will try - I'm already doing that to OE, AE, IJ and such digrams. Maybe grouping the numbers, i already grouped some Greek letters - Maybe grouping the brackets equals and such symbols. The more Letter to Letter we make disappear the more the other categories suffer.

**Comment by VivaRado** [comment]
At the moment (I will be updating this message with the reductions):

{'GG': 3845, 'GL': 1633, 'LG': 1386, 'LL': 484}
Total Pairs :boom: 7348 Now, ( < 10462 < 12000 < 16000 ) Before

**Comment by VivaRado** [comment]
If we have Classes like @A1 = [A Aacute AE]; @E = [E Eacute AE]; mixed side classes.. 
then we have exaggerated kerning values on the regular - but normal kerning values on the italics for A V for example. Which is actually not allowed as its being asserted, but I had to suppress it to have mixed side classes.
```
File "/usr/local/lib/python3.6/dist-packages/fontTools/ufoLib/__init__.py", line 432, in readGroups
    raise UFOLibError(message)
fontTools.ufoLib.errors.UFOLibError: The glyph "AE" occurs in too many kerning groups.
```
we have kerning table overflow, BUT we also have kerning interpolation between reg and it! hmm.jpeg
And not always, the compile is not consistent. It either compiles and ignores the exaggeration, interpolates only the exaggeration, or interpolates the exaggerated and normal value?????

Also depends on ```<kerning copy="1" />``` added to the Italics in the designspace file

I guess this is the support on version 3.21.1:
3.21.1 (released 2018-01-03)
    [feaLib] Allow mixed single/multiple substitutions (#612)
but fontmake uses version 3
